private-channels.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: channel
+title: Private Channels
+permalink: /private-channels
+---
+
+## Overview
+
+Over time a few private channels (either invite only, or invite on request) have sprung up. These are usually formed around a topic that is verboten for public discussion, though not illegal or otherwise banned.
```

Comment:
personally I think 

> that is verboten for public discussion, though not illegal or otherwise banned.

is a bit confusing what does 'verboten but not banned' mean? 

Maybe "These are often formed around more controversial or divisive topics, and want to avoid the oversight that the moderation team carries out in public channels."

Reply:
Much better, thanks!